src/Facebook/InstantArticles/Transformer/Rules/Rule.php

```diff
@@ -23,9 +23,8 @@ public function matches($context, $node)
             $log->debug('node matches: '.($matches_node ? 'MATCHES' : 'no match'));
             $log->debug('rule: '.get_class($this));
             $log->debug('-------');
-            return true;
         }
-        if ($node->nodeName === 'img') {
+        if ($node->nodeName === 'iframe') {
```

Comment:
I actually vote for removing these debug logs we use for when we are developing a rule from the code altogether. For instance, you are here replacing the log for image with a log for iframe just because it was useful for you when developing this PR, but will hardly be useful to anyone moving forward.

Let's make the code more clean and readable by removing these huge blocks of logs.